src/chrome/content/rules/lezhin.com.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="lezhin.com">
+	<target host="lezhin.com" />
+	<target host="www.lezhin.com" />
+
+	<rule from="^http://lezhin.com" to="https://www.lezhin.com" />
```

Comment:
The `from` attribute is regex, you need to escape the dot.
Please also add a comment, that ^lezhin.com is refused over https.